sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/FormRecognizerClientTestBase.java

```diff
@@ -527,14 +557,63 @@ String getApiKey() {
     protected String getEndpoint() {
```

Comment:
I saw this method twice in this PR.  Do we want to put it in TestUtil?

Reply:
Cannot move them outside of the TestBase classes as they the `interceptorManager` class instance to find the corresponding configurations.